### PR TITLE
Added footer to the website like creativecommons.org

### LINKED ISSUE #34: Add a Footer to the page
## Problem

Currently, the bottom of the page looks a bit odd. 

## Description

Adding a footer there will improve the look and feel of the page and it will also match the  current CC aesthetics

## Implementation

- [x] I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by Murdock9803**
## Fixes

Fixes #34 by @tinniaru3005

## Description

The PR adds a footer to the page, just like the one in [creativecommons.org](https://www.creativecommons.org) website.

## Technical details

Added footer section in `listing.html`, added css in `style.css` and included `fonts, SVGs` and new file `vocabulary.css`. Followed the code scheme of the main CC website, assigning variables to colors and svg.

## Screenshots

Screenshot 2024-03-11 064909

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by Murdock9803** [comment]
@possumbilities @Xaid-vfx 
Here is the revised PR for the addition of footer to the website. please have a look for any issues with the same 🙌

**Comment by possumbilities** [comment]
@Murdock9803 Thanks for these changes, there's just a few more things. To pull this off you should add the entire `vocabulary` directory in to the /docs folder? or where relevant. 

Then items like the css, assets, js, fonts will all work if they're included properly. 

See the comment I made on a similar PR. vocabulary is more or less an "all or nothing" inclusion from a dependency standpoint.
* https://github.com/creativecommons/cc-resource-archive/pull/198#issuecomment-1985883117

and since adding vocabulary completely is required to implement the footer, it would also open up the ability to add other areas (like the new header) in future PRs.

**Comment by Murdock9803** [comment]
@possumbilities I have done the required changes, Although the footer worked perfectly with all the fonts in previous iteration of this PR,  but adding the new `vocabulary` directory opens up possibilities for future enhancements in header and other areas too (as advised by you). 
So, I added the whole directory in the `/docs` folder and edited the paths to work fine 🙌 
Please suggest any changes if required to be made 🙌

File: docs/_layouts/listing.html [hunk 1]
```diff
@@ -136,6 +136,73 @@ <h1>
 </div>
 
 
+<!-- Footer -->
+<footer>
+    <a class="identity-logo" href="https://www.creativecommons.org">Creative Commons</a>
+
+    <nav class="footer-menu">
```
Comment (possumbilities) [inline docs/_layouts/listing.html:143]: the links in this menu should have absolute links to the relevant pages on CreativeCommons.org.

File: docs/style.css [hunk 2]
```diff
@@ -257,4 +261,12 @@ width: 210px;
...
 margin-left: 100px;
+}
+
+footer a {
+    color: var(--vocabulary-brand-color-turquoise);
+}
+
+footer .donate a {
+    color: #000;
 }
```
Comment (possumbilities) [inline docs/style.css:272]: this doesn't have an End of File line.

**Comment by possumbilities** [comment]
@Murdock9803 Thank you! I add two small comments of correction and I *think* that should wrap this PR for approval. Great Work! ❤️

File: docs/_layouts/listing.html (hunk 1 above)
Comment (possumbilities) [reply docs/_layouts/listing.html:143]: For example instead of `/about`, it would have to be something like: `https://creativecommons.org/about`

**Comment by Murdock9803** [comment]
@possumbilities I have added the correct absolute links to the footer and also the End-of-line line in style.css. I should've kept these in mind while making the PR, and will surely try to make more perfect ones 🙌

**Review by possumbilities — APPROVED** [review]
Great work, Approved!

**Comment by Murdock9803** [comment]
It feels great when the work is successfully merged 🙌 😄

**Pull request merged**